frontend/packages/console-dynamic-plugin-sdk/docs/console-extensions.md

```diff
@@ -1038,7 +1038,7 @@ This extension can be used to add a tab on the resource details page.
 | Name | Value Type | Optional | Description |
 | ---- | ---------- | -------- | ----------- |
 | `model` | `ExtensionK8sKindVersionModel` | no | The model for which this provider show tab. |
-| `page` | `{ name: string; href: string; }` | no | The page to be show in horizontal tab. It takes tab name as name and href of the tab |
+| `page` | `{ name: string; href: string; }` | no | The page to be show in horizontal tab. It takes tab name as name and href of the tab.<br/>Note: The href should not include forward slashes. |
```

Comment:
Hey, @rhamilto. I put this comment in the JIRA. What do you think?

> I'd suggest we URL encode any special characters in href with encodeURIComponent and always treat it as a single path element. It's a low risk change, and that way things work for any existing plugins passing special characters. It does mean that any existing bookmarks to the tab would be broken, but that's better than the tab navigation breaking. Then we can document the behavior in the API docs.

Reply:
Never mind.  I see there are more changes to make it work.  Working through that now.